### PR TITLE
html and css linters 

### PR DESCRIPTION
**Pull request by Masouma96**
Html and CSS lLinters

### CODE REVIEW

**Comment by Masouma96** [comment]
github action
linter for html
linter for css

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
+
+<body>
+    <h1>Hello Microverse</h1>
+    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptas laudantium possimus reiciendis pariatur
+        voluptatum facilis, quidem tempore dolorem illum molestiae suscipit dolore esse voluptates fugiat doloribus?
+        Quod ab doloremque sint.</p>
+</body>
```
Comment (Nemwel-Boniface) [inline index.html:17]: - [ ] According to the project requirements, you are required to have a ```<h1>``` and not include the <p> tag. Kindly remove the ```<p>``` tag.
- [ ] Also give your ```<h1>``` a class 

image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    float: left;
+    padding-left: 10px;
+    align-content: center;
+}
+p{
+    color: black;
+    float: left;
+    
+
+}
```
Comment (Nemwel-Boniface) [inline style.css]: - [ ] Kindly fix the syntax errors that can be visible in your.css file. To auto-fix the linters run ```npx stylelint "**/*.{css,scss}" --fix```

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @masouma2020           :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done an amazing job with your project so far :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [x] Kindly make your README.md file descriptive. Currently, I can see that it has the boilerplate and does not say much about what the project is about.
- [x] Also,in your .gitignore I cannot see the node_modules ignored. Kindly ignore it in your .gitignore

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**